wrapanapi/systems/redfish.py

```diff
@@ -103,7 +139,9 @@ class RedfishSystem(System):
 
     # statistics for the provider
     _stats_available = {
-        'num_server': lambda self: self.num_servers,
+        'num_server': lambda server: server.num_servers,
```

Comment:
Nitpicking, but

```suggestion
        'num_server': lambda system: system.num_servers,
```

(same for the two lines below)

Because those functions are defined on RedfishSystem, not RedfishServer

Reply:
Well-spotted! 👍

Reply:
This still doesn't look right to me - `RedfishChassis.num_servers` is where the method is defined, @miha-plesko referenced `System.num_servers`, but the diff still shows `server.num_servers` call?

Reply:
Well actually the diff shown above is my "suggestion", a new feature of GitHub. You can try it in your comments with this button here:

![capture](https://user-images.githubusercontent.com/8102426/48420337-77560880-e75a-11e8-8103-23c0cb4e5c3e.PNG)

I see it just introduced confusion, sorry about that. As you say @mshriver the comment is still vaild - lambda argument should be renamed to 'server'.

Reply:
Additionally, it looks like the two calls below to `num_chassis` and `num_racks` are incorrect as well?  Both of those methods are defined on the local RedfishSystem class, and should be called with `self.num_<thing>` yes?

Reply:
I really apologize for the confusion - this too was stuck in my un-pushed change. I belive that now it is as it should be - the `_stats_available` referring to `system`. 

As for the `rack` parameter in `_rack_*_available` and `chassis` parameter in `_chassis_*_available`, they get obtained through `get_rack` and `get_chassis`, respectively, so the properties are implemented in `RedfishChassis` (which represents both the Redfish racks and the Redfish chassis at the API end). The integration tests use the `RedfishSystem` just to get to racks, chassis, servers, etc., but the definition for the `num_<thing>`s is in their respective specialized classes.